### ISSUE TITLE
The model and loaded state dict do not match exactly

### ISSUE THREAD

**Issue by xukaiquan** [open-mmlab/mmfashion#53]
Dear Authors,

When I try the landmark detector demo, I meet the following issues. Please give one good demo or example. Thanks

pretrained model checkpoint/vgg16.pth
The model and loaded state dict do not match exactly

unexpected key in source state_dict: classifier.0.weight, classifier.0.bias, classifier.3.weight, classifier.3.bias, classifier.6.weight, classifier.6.bias

model built
load checkpoint from: checkpoint/LandmarkDetect/global/landmark_detect_best.pth
F:\Python\Anaconda3\envs\cuhk_mmlab\lib\site-packages\torch\nn\functional.py:1569: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
detected landmark [ 49.13241 134.01773] [169.72472 133.90952]
detected landmark [169.72472 133.90952] [ 67.13259 172.5399 ]
detected landmark [ 67.13259 172.5399 ] [151.30598 170.36455]

**Comment by veralauee** [comment]
It already load the right model.
Note that the model is firstly initialized via "vgg16.pth" where it comes up with some unexpected keys. But after the model built, it "load checkpoint from: checkpoint/LandmarkDetect/global/landmark_detect_best.pth". 
The model is loaded successfully.

**Comment by Wangnaijia** [comment]
Hello, I wonder if i have downloaded the right model? The result of landmark detector seems poor, here I used the vgg16 model downloaded from baidu drive, and got only three landmarks on the landmark_predict/demo1.jpg. How can I get better detection like shown in the gif?

python demo/test_landmark_detector.py --input demo/imgs/landmark_predict/demo1.jpg
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda-9.0'
pretrained model checkpoint/vgg16.pth
The model and loaded state dict do not match exactly

unexpected key in source state_dict: classifier.0.weight, classifier.0.bias, classifier.3.weight, classifier.3.bias, classifier.6.weight, classifier.6.bias

model built
load checkpoint from: checkpoint/LandmarkDetect/global/landmark_detect_best.pth
/home/wnj/venvs/Pytorch/lib/python3.6/site-packages/torch/nn/functional.py:1351: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
detected landmark 36.98792670454298 165.01900127955847
detected landmark 166.21276821408955 165.32434395381384
detected landmark 55.93586351190295 166.0515904426575

**Comment by xinzi2018** [comment]
i have the same question!!!

**Comment by FeatureSpitter** [comment]
This project is unusable, what a pitty.
